client/src/pages/SignUp.js

```diff
@@ -0,0 +1,178 @@
+                            Enter your e-mail to get started:
+                            </Typography>
+                        <BootstrapInput fullWidth id="email" placeholder="E-mail address"
+                            type="email" variant="outlined" className={classes.input}
+                            onChange={handleChange} value={email} />
+                        <Button className={classes.btnOrange}
+                            onClick={handleGetStarted}>Get Started</Button>
+                    </CardContent>
+                    <Divider light />
+                    <CardContent>
+                        <small className={classes.textFooter}>
+                            Already have an account?&nbsp;
+                                    <a href="/login" id="login"
+                                className={classes.textOrange}>
+                                Log in
+                                    </a>
+                        </small>
+                    </CardContent>
+                </Grid>
+            )}
+            {confirmed === true && (
+                <Grid item className={classes.shadowCard}>
+                    <CardContent className={classes.body}>
+                        <Typography variant="h6" className={classes.title}>
+                            Hi {email}!
+                            </Typography>
+                        <Typography variant="body2" color="textSecondary" component="p"
+                            className={classes.description}>
+                            The easiest way for you to sign up is with Google.
+                                <br />This will automatically connect your calendar
+                                so you can start using CalendApp right away!
+                            </Typography>
+                        <CardActions>
+                            <Button className={classes.btnOrange}>
+                                <svg aria-hidden="true" focusable="false" height="16" width="16" data-prefix="fab" data-icon="google" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 488 512" class="svg-inline--fa fa-google fa-w-16 fa-5x"><path fill="currentColor" d="M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z" class=""></path></svg>
+                                    &nbsp;&nbsp;&nbsp;&nbsp;Sign up with Google
+                                </Button>
+                        </CardActions>
+                    </CardContent>
+                    <Divider light />
+                    <CardContent>
+                        <small className={classes.textFooter}>
+                            Prefer to create an account with a password?
+                                <br /><a href="/Signup" id="signup"
```

Comment:
Use `Typography` for text. `react-router-dom` [Link](https://reactrouter.com/web/api/Link) component for linking